README.md

```diff
@@ -20,9 +20,9 @@ the full power of your database.
 
 Main rom gem provides following components:
 
-* [core](https://github.com/rom-rb/rom/blob/master/core/README.md) - Core and Adapter APIs
-* [changeset](https://github.com/rom-rb/rom/blob/master/changeset/README.md) - Changeset objects integrated with rom-core
-* [repository](https://github.com/rom-rb/rom/blob/master/repository/README.md) - Additional repository abstraction integrated with rom-core
+* [core](https://github.com/rom-rb/rom/blob/master/docsite/core/source/index.html.md) - Core and Adapter APIs
+* [changeset](https://github.com/rom-rb/rom/blob/master/docsite/changeset/source/index.html.md) - Changeset objects integrated with rom-core
+* [repository](https://github.com/rom-rb/rom/blob/master/docsite/repository/source/index.html.md) - Additional repository abstraction integrated with rom-core
```

Comment:
We could just point to https://rom-rb.org/learn/{core,changeset,repository/5.2/ pages instead of sources in the git repo

Reply:
I've adapted the links as you suggested. But I am wondering if they should include `5.2/` (and then have to be updated in the README every time) or if they should just point to the latest version of the docs?

Reply:
@mediafinger it's fine for now. We need to update the website in general anyway.